packages/qocamlbrowser/qocamlbrowser.0.2.10/opam

```diff
@@ -18,12 +18,12 @@ remove: [
   [make "uninstall"]
 ]
 depends: [
-  "ocaml" {>= "4.04.0"}
+  "ocaml" {>= "4.07.0"}
```

Comment:
why this change? It seemed to work ok in https://github.com/ocaml/opam-repository/pull/12384